catalogue/webapp/services/prismic/fetch/index.ts

```diff
@@ -0,0 +1,90 @@
+import * as prismic from '@prismicio/client';
+import fetch from 'node-fetch';
+import { gql, GraphQLClient } from 'graphql-request';
+import { PrismicApiError } from '../types';
+import { capitalize } from '@weco/common/utils/grammar';
+
+export type GetServerSidePropsPrismicClient = {
+  type: 'GetServerSidePropsPrismicClient';
+  client: prismic.Client;
+};
+
+export const typesToPrismicGraphQLSchemaTypes = {
+  // types to graphql query schema types,
+  events: 'allEventss',
+  exhibitions: 'allExhibitionss',
+  articles: 'allArticless',
+  series: 'allSeriess',
+  webcomics: 'allWebcomicss',
```

Comment:
Oh, question, should we consider series and webcomics as part of "stories"? Still wrapping my head around that.

Reply:
Yep, I was starting the work to get these in when I looked at the original content/prismic fetch function for articles

Reply:
Very good shout. As per our chat and the work on 'overview' ticket, I'm going to tackle this in another PR.